proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyImpl.java

```diff
@@ -33,8 +33,8 @@ protected HttpClient newHttpClient() {
     sslFactory.setSslSessionTimeout((int) TimeUnit.SECONDS.toMillis(15));
 
     HttpClient httpClient = new HttpClient(sslFactory);
-    httpClient.setMaxConnectionsPerDestination(200); // overriding default 64
-    httpClient.setConnectTimeout(TimeUnit.MINUTES.toMillis(1));
+    httpClient.setMaxConnectionsPerDestination(2000);
+    httpClient.setConnectTimeout(TimeUnit.SECONDS.toMillis(30));
```

Comment:
Use ENUMs or a config

Reply:
This is one time config change. Underlying library also declares these inline (improves readability) which is set to `256` in the version we are using. 
The example below has set it `Integer.MAX_VALUE`

https://android.googlesource.com/platform/external/jetty/+/refs/heads/marshmallow-dev/src/java/org/eclipse/jetty/client/HttpClient.java#82